src/gz390.java

```diff
@@ -3148,7 +3148,7 @@ private String get_ascii_string(byte[] text_byte,int lbuff){
 	}
 	return text;
 }
-     /***********************************************
+     /* *********************************************
```

Comment:
If this is supposed to be javadoc, shouldn't it be "/** ********************************"??? That is, "/**". followed by one space, followed by rest of asterisks?